packages/presentation/src/components/ObjectPopup.svelte

```diff
@@ -72,7 +73,7 @@
     (result) => {
       objects = result
     },
-    { ...(options ?? {}), limit: 200 }
+    { limit: 200, ...(options ?? {}) }
```

Comment:
Bad idea, we should not allow to override limit

Reply:
Maybe we should use less limit?

Reply:
Reverted.